### PR TITLE
Edits to Gen 2 project structure page

### PR DESCRIPTION
**Pull request by hibler13**
### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [ ] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] Swift
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [ x] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ x] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br /> 
      _ref: MDX: `[link](https://docs.amplify.aws/)` 
            HTML: `<a href="https://docs.amplify.aws/">link</a>`_

### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/gen2/reference/project-structure/index.mdx [hunk 1]
```diff
@@ -101,7 +101,7 @@ defineBackend({
 });
 ```
 
-Backends can be extended using the [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/v2/guide/home.html). By leveraging CDK you can build using any AWS service, such as an Amazon S3 bucket that authenticated users have read and write access to. You can get started by installing AWS CDK:
+You can extend backends by using the [AWS Cloud Development Kit (AWS CDK)](https://docs.aws.amazon.com/cdk/v2/guide/home.html). With the CDK, you can build using any AWS service, such as an Amazon S3 bucket that authenticated users have read and write access to. To get started with the CDK, first install it:
```
Comment (josefaidt) [inline src/pages/gen2/reference/project-structure/index.mdx:104]: should we use "AWS CDK" when referring to CDK?

Comment (hibler13) [reply src/pages/gen2/reference/project-structure/index.mdx:104]: According to the AWS service names wiki, we should spell out "AWS Cloud Development Kit (AWS CDK)" on first use. It's OK to use the extra-short name ("CDK") in service documentation when the extra-long name ("AWS Cloud Development Kit") or short name ("AWS CDK") has already been used in the same topic.

Comment (josefaidt) [reply src/pages/gen2/reference/project-structure/index.mdx:104]: that makes sense to me! resolving

**Review by josefaidt — DISMISSED** [review]

**Comment by hibler13** [comment]
@josefaidt I've made the requested changes. Can you review again? Thanks!

**Review by josefaidt — APPROVED** [review]

**Review by heatheramz — APPROVED** [review]
LGTM

**Pull request merged**